src/Symfony/Component/Cache/Tests/Adapter/MemcachedAdapterTest.php

```diff
@@ -248,4 +249,36 @@ public function testMultiServerDsn()
         ];
         $this->assertSame($expected, $client->getServerList());
     }
+
+    public function testDoubleEncodingBug()
+    {
+        $reservedMemcachedCharacters = " \n\r\t\v\f\0";
+
+        $namespace = ItemInterface::RESERVED_CHARACTERS.$reservedMemcachedCharacters.random_int(0, \PHP_INT_MAX);
```

Comment:
namespaces cannot contain reserved characters: that's a restriction that is shared by all adapters, no reason to make this one different

Reply:
@nicolas-grekas fixed